src/test/java/org/folio/rest/impl/OverdueFinePoliciesAPITest.java

```diff
@@ -142,10 +144,42 @@ public void postOverdueFinePolicyWithZeroFineAmount() {
       .contentType(ContentType.JSON);
   }
 
+  @Test
+  public void postOverdueFinePolicyWithNegativeQuantity() {
+    JsonObject entityJson = createEntityJson();
+    entityJson.getJsonObject("overdueFine").put("quantity", -1);
+    entityJson.getJsonObject("overdueRecallFine").put("quantity", -2);
+
+    post(entityJson.encodePrettily())
+      .then()
+      .statusCode(HttpStatus.SC_UNPROCESSABLE_ENTITY)
+      .body(containsString(NEGATIVE_QUANTITY_MESSAGE))
+      .contentType(ContentType.JSON);
+  }
+
+  @Test
+  public void putOverdueFinePolicyWithNegativeQuantity() {
+    JsonObject entity = createEntityJson();
+    post(entity.encodePrettily());
+
+    entity.getJsonObject("overdueFine").put("quantity", "-1");
+    entity.getJsonObject("overdueRecallFine").put("quantity", "-1");
```

Comment:
Split into two tests. Why here values are strings?

Reply:
Because I saw that kind of request are sent from UI

Reply:
done

Reply:
> Anyway on the back-end side it doesn't matter
This is a bug. Backend should care about data type.